README.md

```diff
@@ -113,16 +119,23 @@ The tool should now be installed as `$GOPATH/bin/harbourbridge`
 To use the tool on a PostgreSQL database called mydb, run
 
 ```sh
-pg_dump mydb | $GOPATH/bin/harbourbridge
+pg_dump mydb | harbourbridge -driver=pgdump
 ```
 
-This command will use the cloud project specified by the GCLOUD_PROJECT
+To use the tool on a MySQL database called mydb, run
+
+```sh
+mysqldump mydb | harbourbridge -driver=mysqldump
+```
+
+More details on running harbourbridge can be found in [Example usage](#example-usage) section.
+
+This command will use the cloud project specified by the `GCLOUD_PROJECT`
```

Comment:
we should add this line somewhere here:

"This command will use the cloud project specified by the GCLOUD*PROJECT
environment variable, automatically determine the Cloud Spanner instance associated with this project. If there is only one spanner instance running in the project, that instance will be selected, if there are multiple instances available then list of available instances will be shown and you have to run the command again with `-instance=<instance-name>` flag."

Reply:
Agreed. This should be added.